docs/benchmarking.md

```diff
@@ -90,6 +95,122 @@ perf_analyzer -m facebook/opt-125m --service-kind=openai --endpoint=v1/chat/comp
 # Concurrency: 1, throughput: 4.94426 infer/sec, latency 200467 usec
 ```
 
+## Benchmarking OpenAI Multi-Turn Chat
+
+Here is an example of using Perf Analyzer to benchmark a multi-turn chat model
+on an OpenAI API-compatible server:
+
+> [!NOTE] Multi-turn chat benchmarking does not report any statistics. It is
+> only intended to output a trace of the benchmark
+> ([`--profile-export-file`](cli.md#--profile-export-file-path)), which is used
+> by [GenAI-Perf](../genai-perf/docs/multi_turn.md) to calculate benchmark
+> statistics.
+
+<details>
+<summary>See instructions</summary>
```

Comment:
Since this example is less meaningful for the PA user (relies on GAP to produce benchmark statistics), I've hidden this example in a details element.